inorbit_edge/robot.py

```diff
@@ -469,6 +470,36 @@ def report_command_result(self, args, execution_id, result_code):
         msg.return_code = result_code
         self.publish_protobuf(MQTT_SCRIPT_OUTPUT_TOPIC, msg)
 
+    def register_executable_commands(
+        self, exec_name_regex=r".*\.sh", path="./user_scripts"
+    ):
+        """Registers executable commands that handle InOrbit script actions.
```

Comment:
I'd clarify a bit what kind of actions will get here.
Maybe say: actions of type "Run Script on Agent"? (that's how it looks like in the config) 
It may also be worth mentioning that other type of actions are ignored.